components/burger/Burger.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Story, Meta } from "@storybook/react/types-6-0";
+import Burger from "../burger/Burger";
+import { BurgerProps } from "../burger/Burger";
+
+export default {
+  title: "Common/Burger",
+  component: Burger,
+} as Meta;
+
+const Template: Story<BurgerProps> = (args) => <Burger {...args} />;
+
+export const Closed = Template.bind({});
+Closed.args = { open: false };
+
+export const Open = Template.bind({});
+Open.args = { open: true };
```

Comment:
it is necessary to say true or false or can you set an boolean here?

Reply:
If I understand it correctly this is the start attribute of the boolean inside of Burger.tsx